docs/tutorials/gstreamer.mdx

```diff
@@ -1 +1,100 @@
+---
+sidebar_position: 4
+---
 
+# Build GStreamer pipeline with opencv-python 
+
+This tutorial demonstrates how to use OpenCV with GStreamer to capture video from cameras. It supports different camera configurations, pixel formats, and frame size.
+
+## Tutorial
+
+### Prerequisites
+1. opencv-python with gstreamer backend
+3. **windows**:  
+    Install the C++ version of sensing-dev. make sure GST_PLUGIN_PATH is set.
+3. **linux**:  
+   Set the GST_PLUGIN_PATH environment variable and sensing-dev library_path:
+   ```bash
+   export GST_PLUGIN_PATH=/opt/sensing-dev/lib/x86_64-linux-gnu/gstreamer-1.0
+   export LD_LIBRARY_PATH=/opt/sensing-dev/lib:/opt/sensing-dev/lib/x86_64-linux-gnu:$LD_LIBRARY_PATH
+   ```
+
+### Configuration
+Please adjust the camera_name, pixelformat, width, height, framerate based on your requirements
```

Comment:
can you mention that you can obtain these values are obtained by arv-tool-0.8? Reference: https://github.com/Sensing-Dev/doc/blob/027fc72487ddedc54bc2a74cfaf279f68973919f/docs/lessons/camera.md?plain=1#L26